airbyte-cdk/java/airbyte-cdk/core/src/main/resources/version.properties

```diff
@@ -1 +1 @@
-version=0.47.0
+0.47.1
```

Comment:
@theyueli in my latest merge I mistakenly removed the key `version`. Apologies for that. I have corrected this here https://github.com/airbytehq/airbyte/pull/46101 🙏